crates/rnote-ui/data/ui/dialogs/dialogs.ui

```diff
@@ -203,4 +203,15 @@ Changes which are not saved will be permanently lost.</property>
       <object class="RnIconPicker" id="edit_selected_workspace_icon_picker"></object>
     </child>
   </object>
+
+  <object class="AdwAlertDialog" id="dialog_trash_file">
+    <property name="heading" translatable="yes">Trash File</property>
+    <property name="body" translatable="yes">Are you sure you want to delete this file?</property>
+    <property name="default-response">trash</property>
```

Comment:
The default selected response should be to cancel

Reply:
“This property determines what happens when the Return key is pressed while the alert is shown.”, on enter any file manager or application I tested always confirms deletion, and this is also the behaviour I would expect. Because otherwise, how does one confirm without mouse input? On escape, however, we definitely would want to cancel.

Reply:
At least the Gnome [Human Interface Guidelines](https://developer.gnome.org/hig/patterns/feedback/dialogs.html#dialogs) suggest to make cancelling the default for destructive actions (last paragraph).

Although admittedly Rnote is already not following that consistently with the "Clear Document" button.

The keyboard focus should be on the default action button so it would take an additional right nav-key

Reply:
Okay, fair enough, it's your call, I don't have an issue with changing it. It is however weird because it's the same for Nautilus. If you shift + del to permanently delete something and press enter, it confirms the deletion.

Reply:
I don't have a strong opinion about it either but I usually stick to the HIG . I'll change the `Clear Document` action to it as well.

Curious that Nautilus as the core file mangement app doesn't do that - especially when the action actually is permanent and possibly very destructive.

Reply:
Alright, I'll change it as well. I agree, I wasn't aware that's how the HIG handles it, I just thought about the default behaviour in file managers and stuck to that. But it's changed now :)

Reply:
Alright, thanks :)